From cd50aae0cd4ffc5a7b212b10f5f78ef4c3a7e915 Mon Sep 17 00:00:00 2001
From: Fugang Duan <fugang.duan@nxp.com>
Date: Tue, 30 Jan 2018 15:53:55 +0800
Subject: [PATCH 3290/5242] MLK-17475-04 net: phy: tja110x: clean up the debug
 error message

commit  5dedefc1d80c0193f550983d002f32b99200e6ac from
https://source.codeaurora.org/external/imx/linux-imx.git

Some code slices have handled the code logic in correctly, it should
not print out the error message. So clean up them.

Reviewed-by: Pandy Gao <pandy.gao@nxp.com>
Signed-off-by: Fugang Duan <fugang.duan@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/net/phy/tja110x.c |   12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/drivers/net/phy/tja110x.c b/drivers/net/phy/tja110x.c
index 9ec978b..5134cec 100644
--- a/drivers/net/phy/tja110x.c
+++ b/drivers/net/phy/tja110x.c
@@ -726,7 +726,7 @@ static int nxp_sleep(struct phy_device *phydev)
 
 /* error handling */
 phy_auto_op_error:
-	dev_err(&phydev->mdio.dev, "phy is in auto mode: sleep not possible\n");
+	dev_info(&phydev->mdio.dev, "phy is in auto mode: sleep not possible\n");
 	return 0;
 
 phy_configure_error:
@@ -826,7 +826,7 @@ static int wakeup_from_sleep(struct phy_device *phydev)
 
 /* error handling */
 phy_auto_op_error:
-	dev_err(&phydev->mdio.dev, "phy is in auto mode: wakeup not possible\n");
+	dev_dbg(&phydev->mdio.dev, "phy is in auto mode: wakeup not possible\n");
 	return 0;
 
 phy_configure_error:
@@ -936,7 +936,7 @@ static int nxp_suspend(struct phy_device *phydev)
 	return err;
 
 phy_auto_op_error:
-	dev_err(&phydev->mdio.dev, "phy is in auto mode: suspend not possible\n");
+	dev_dbg(&phydev->mdio.dev, "phy is in auto mode: suspend not possible\n");
 	return 0;
 }
 
@@ -1485,7 +1485,7 @@ static ssize_t sysfs_set_loopback_cfg(struct device *dev,
 
 /* error handling */
 phy_auto_op_error:
-	dev_err(&phydev->mdio.dev, "phy is in auto mode: loopback not available\n");
+	dev_info(&phydev->mdio.dev, "phy is in auto mode: loopback not available\n");
 	return count;
 
 phy_parse_error:
@@ -1549,7 +1549,7 @@ static ssize_t sysfs_get_cable_test(struct device *dev,
 
 /* error handling */
 phy_auto_op_error:
-	dev_err(&phydev->mdio.dev, "phy is in auto mode: cabletest not available\n");
+	dev_info(&phydev->mdio.dev, "phy is in auto mode: cabletest not available\n");
 	return 0;
 
 phy_read_error:
@@ -1686,7 +1686,7 @@ static ssize_t sysfs_set_test_mode(struct device *dev,
 
 /* error handling */
 phy_auto_op_error:
-	dev_err(&phydev->mdio.dev, "phy is in auto mode: testmodes not available\n");
+	dev_info(&phydev->mdio.dev, "phy is in auto mode: testmodes not available\n");
 	return count;
 
 phy_parse_error:
-- 
1.7.9.5

